Mage/src/main/java/mage/game/permanent/token/WizardBlackToken.java

```diff
@@ -0,0 +1,38 @@
+package mage.game.permanent.token;
+
+import mage.MageInt;
+import mage.abilities.common.SpellCastControllerTriggeredAbility;
+import mage.abilities.effects.common.DamagePlayersEffect;
+import mage.constants.CardType;
+import mage.constants.SubType;
+import mage.constants.TargetController;
+import mage.filter.StaticFilters;
+
+/**
+ * @author balazskristof
+ */
+public final class WizardBlackToken extends TokenImpl {
```

Comment:
I already made this token as `BlackWizardToken` for Circle of Power so just delete that one and update the card to use this one since I know you said you were working on this a few days ago